interface/patient_file/summary/demographics_save.php

```diff
@@ -183,4 +183,6 @@
         filter_input(INPUT_POST, 'i3policy_type')
     );
 }
- include_once("demographics.php");
+
+// if refresh tab after saving then results in csrf error
+include_once("demographics.php");
```

Comment:
minor bug here is the tab is still set to `demographics_save` even though switched context to demographics. php, not sure how to address this